### PR TITLE
Prepare Release 1.6

### PR DESCRIPTION
**Pull request by sj14**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Deprecated features or components might be removed in later versions without cha
 
 ### 1.next
 
+### 1.6.0
+
+- Add bird-configwatcher
```
Comment (jellonek) [inline CHANGELOG.md:13]: Please also add info about https://github.com/openvnf/cgw/pull/57

Comment (sj14) [reply CHANGELOG.md:13]: Thanks, added info about rclone indentation and security context.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ in the current version and therefore not recommended for production use.
   * [BGP](#bgp)
     * [BIRD Internet Routing Daemon](#bird-internet-routing-daemon)
     * [bird_exporter](#bird_exporter)
+    * [bird_exporter](#bird-configwatcher)
```
Comment (hudymi) [inline README.md:21]: Copy-paste issue ;)

Comment (sj14) [reply README.md]: Thanks. Fixed in https://github.com/openvnf/cgw/pull/60/commits/0eca503d51669cb7a7ee01ca4d99688f6418e3fd

**Review by hudymi — APPROVED** [review]

**Pull request merged**